elliottlib/cli/generate_ccs_image_list_cli.py

```diff
@@ -0,0 +1,31 @@
+import click
+
+from elliottlib import Runtime
+from elliottlib.cli.common import cli, find_default_advisory
+from elliottlib import ccs_image_list
+
+pass_runtime = click.make_pass_decorator(Runtime)
+
+
+@cli.command('generate-ccs-image-list', short_help='Generate image lists for Customer Content Services')
+@click.option(
+    '--advisory', '-a', 'advisory',
+    default=None, metavar='ADVISORY',
+    help='Explicitly define image advisory ID to be used instead of the default')
+@pass_runtime
+def generate_ccs_image_list_cli(runtime, advisory):
+    """Generate the image list of a given advisory for CCS (docs team).
+
+    $ elliott --group openshift-4.2 generate-ccs-image-list --advisory 48465
```

Comment:
typically need it in case you want to look up the advisory based on group. even if not using it in some use cases IIRC a subcommand can't get a runtime at all without requiring it always.

Reply:
@sosiouxme `Runtime` 's `no_group` option has been fixed. Sure if calling other subcommands that will be a problem. This is not required, just something nice to have.

Reply:
done, now `--group` can be omitted if `--advisory` is present:
https://github.com/openshift/elliott/pull/90/files#diff-81648ee0ffacce9456b9b0273306dcdaR20
https://github.com/openshift/elliott/pull/90/files#diff-81648ee0ffacce9456b9b0273306dcdaR27